include/devicetree.h

```diff
@@ -45,6 +45,7 @@
  * _ENUM_UPPER_TOKEN: like _ENUM_TOKEN, but uppercased
  * _EXISTS: property is defined
  * _FOREACH_PROP_ELEM: helper for "iterating" over values in the property
+ * _VARGS: foreach functions with variable number of arguments
```

Comment:
This needs to be the complete suffix instead, i.e. `_FOREACH_PROP_ELEM_VARGS`. It's the way we make sure there are no namespace collisions.

Reply:
Understood. 

Can you please take a look at the build errors regarding DT_CAT below and advise if it would be ok for me to revert the DT_CAT changes, or if there is another way to fix the issue.

Reply:
I think I see what's going on and a fix is out of scope. Let's just use UTIL_CAT for now and I'll try to change it in a follow-up PR at some point.